src/MB_Toolbox.php

```diff
@@ -335,6 +335,16 @@ public function curlPOST($curlUrl, $post, $isAuth = FALSE) {
         )
       );
     }
+    elseif (strpos($curlURL, 'api.dosomething') !== FALSE) {
```

Comment:
An interesting note is that we'll likely be using Fastly to route requests to `api.dosomething` to either Northstar or the Drupal content API depending on the type of request. While harmless having these in the headers, I wonder if it'd make sense for your requests to go directly to `northstar.dosomething`.